### PR TITLE
Fixup unused code

### PR DESCRIPTION
**Pull request by iforgotband**

### CODE REVIEW

**Comment by iforgotband** [comment]
Going to remove the travis file now. Inappropriate for the project.

File: js/wNumb.js [hunk 1]
```diff
@@ -23,12 +23,12 @@
...
-	'negativeBefore',
+	"mark",
+	"prefix",
+	"suffix",
+	"encoder",
+	"decoder",
+	"negativeBefore",
 	'negative',
 	'edit',
 	'undo'
```
Comment (DavidLemayian) [inline js/wNumb.js:34]: Any reason to mix single and double quotes?

Comment (iforgotband) [reply js/wNumb.js:34]: Nope. Actually that's a reason to hold off on merger. I'll be cleaning up the singles that are left, that was an oversight!

**Comment by iforgotband** [comment]
A new request will be opened at a later time. This one is messy. Apologies.

**Pull request closed without merging**